### ISSUE TITLE
Adding SGT and S2GT models

### ISSUE THREAD

**Issue by muthumula19** [uber/orbit#668]
LGT and DLT working good and i saw SGT and S2GT model from Rlgt especially for seasonal pattern finding  when i implemented in R they also working good please add these 2 models :)

**Comment by edwinnglabs** [comment]
Good point.  S2GT indeed has some use case for dual seasonalities.  I will consult with Slawek if he is interested in translating it to orbit.
